client/src/index.jsx

```diff
@@ -256,7 +258,16 @@ class App extends React.Component {
             <Route 
               path="/view?=(:id)" 
               render={HomeWithProps}
-              onEnter={ this.requireAuth }/>
+              onEnter={ this.requireAuth }
+              />
```

Comment:
Line closing bracket up with opening bracket

Reply:
This is all over the place in the app. Even in this same file, most of them indent the closing bracket FWIW.